### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amirbenv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,10 @@
 [![community](https://img.shields.io/discourse/status?server=https%3A%2F%2Fcommunity.accurics.com)](https://community.accurics.com)
 [![Documentation Status](https://readthedocs.com/projects/accurics-terrascan/badge/?version=latest)](https://docs.accurics.com/projects/accurics-terrascan/en/latest/?badge=latest)
 
-Detect compliance and security violations across Infrastructure as Code to mitigate risk before provisioning cloud native infrastructure.
+Detect security vulnerabilities and compliance violations across your Infrastructure as Code. Mitigate risks before provisioning cloud native infrastructure. Run locally or integrate with your CI\CD.
 
 
-* GitHub Repo: https://github.com/accurics/terrascan
-* Documentation: https://docs.accurics.com
+* Documentation: https://docs.accurics.com/projects/accurics-terrascan/en/latest/
```
Comment (dev-gaur) [inline README.md:14]:
```suggestion
* Documentation: https://docs.accurics.com/projects/accurics-terrascan
```
/en/latest is redundant.

**Review by dev-gaur — CHANGES_REQUESTED** [review]
one minor change requested. everything else LGTM

File: README.md [hunk 2]
```diff
@@ -109,8 +75,49 @@ Flags:
 Use "terrascan [command] --help" for more information about a command.
 ```
 
-## Documentation
-To learn more about Terrascan check out the documentation https://docs.accurics.com where we include a getting started guide, Terrascan's architecture, a breakdown of it's commands, and a deep dive into policies.
+### Step 3: Integrate with CI\CD
+Please refer to our documentation to integrate with your pipeline:
```
Comment (dev-gaur) [inline README.md:79]: IMHO, this will look better as a hyperlink.

Comment (dev-gaur) [reply README.md:79]:
```suggestion
Please refer to our [documentation to integrate with your pipeline](https://docs.accurics.com/projects/accurics-terrascan/en/latest/cicd/)
```

**Review by dev-gaur — CHANGES_REQUESTED** [review]

**Review by jlk — APPROVED** [review]
LGTM

**Pull request merged**